### ISSUE TITLE
Add Xbox One Bluetooth support

### ISSUE THREAD

**Issue by wyloop** [MrCrayfish/Controllable#404]
I was thinking maybe if there was support for Bluetooth connection for the Xbox One controller, it would be a lot easier for people on different platforms to use it.

**Comment by nttaniel** [comment]
yes!!! im using xbox x controller and i cant use it over bluetooth :((( even xbox one support would be great

**Comment by MegPredator** [comment]
I am pretty sure support depends on your operating system

**Comment by steinermatt** [comment]
Been using my Xbox controller via Bluetooth (on Mac) with Forge and 1.19.4 - and it works great.

**Comment by wyloop** [comment]
no i meant Xbox One not Series X/S

**Comment by MegPredator** [comment]
> no i meant Xbox One not Series X/S

Still should be supported via windows of course
